modules/mvt/src/lib/mapbox-vector-tile/vector-tile-feature.js

```diff
@@ -12,6 +11,7 @@ export default class VectorTileFeature {
     this.properties = {};
     this.extent = extent;
     this.type = 0;
+    this.id = null;
```

Comment:
What is the purpose of the id here?

Reply:
`id` can be set from `readFeature`

Reply:
`readFeature` implicitly adds `id`, but TypeScript complains if the field is never explicitly set.